src/main/java/racingcar/domain/car/DefaultCar.java

```diff
@@ -0,0 +1,38 @@
+package racingcar.domain.car;
+
+import racingcar.domain.numbergenerator.RandomNumberGenerator;
+
+public class DefaultCar implements Car{
+    private static final int MOVE_THRESHOLD = 4;
+
+    private final RandomNumberGenerator numberGenerator;
+    private final String name;
+    private int position;
+
+    public DefaultCar(RandomNumberGenerator numberGenerator, String name) {
+        this.numberGenerator = numberGenerator;
+        this.name = name;
+        this.position = 0;
+    }
+
+
+    @Override
+    public void move() {
+        if (isMovable()){
+            position++;
+        }
+    }
```

Comment:
랜덤 값 생성 로직을 외부로 이동시키면 아래와 같이 변경될 수 있겠네요.

```suggestion
    public void move(int number) {
        if (isMovable(number)){
            position++;
        }
    }
```